web/styles/dark_theme.css

```diff
@@ -293,19 +289,6 @@
         border: 1px solid hsl(11deg 46% 54%);
     }
 
-    .alert {
-        text-shadow: none;
```

Comment:
This isn't part of `close`, right?

Reply:
Correct. Though I believe that `text-shadow` value is unnecessary, as there's nowhere else we're setting `text-shadow` that would necessitate its being turned off here. I'm happy to put this removal in its own commit though.